_posts/2019-02-24-The-Intelligent-Investor.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Book Review: The Intelligent Investor"
+date:   2019-02-24 22:40:05
+categories: jekyll update
+---
+
+### Disclaimer and Basics on Retirement
+
+I am simply an enthusiastic lay person
+who happens to know some math, a fair bit of personal finance,
+and likes reading books on the topic of investing. Please do your own research
+and don't ask me for specific advice about any companies,
+markets, etc.
+
+**I will not help you with individual financial stuff. Please do not ask.** Consult with a fiduciary advisor with any specific question you have and make sure your 401k company is not screwing you over. *Hint*: They are probably screwing you over, watch [The Retirement Gamble](https://www.pbs.org/video/frontline-retirement-gamble/).
+
+
+### The Gist
+
+_The Intelligent Investor_ and _Security Analysis_ is the only investment books I ever needed to read, all the others were further elaboration on Benjamin Graham's points. I decided to do this book review because the principles to preserve wealth, reducing the amount of false positives, and truly invest for the long run (the only investment that exists) are timeless techniques. For the purposes of brevity I will focus primarily on what Benjamin Graham calls a Defensive Investor instead of the Enterprising Investor.
```

Comment:
`For the purposes of brevity I will focus primarily on what Benjamin Graham calls a Defensive Investor instead of the Enterprising Investor.`
- Add a comma after `brevity`. 
- italics (or something) to `Defensive Investor` and `Enterprising Investor`. 
- What are your opinions on adverbs (see `primarily`)? I've heard that they're typically the weakest part of sentences and should be omitted when possible. Here, `focus` somewhat *implies* a limit on scope. So, does this verb need the modifier `primarily`?

Reply:
Fair point on the adverbs, I use them out of habit and remove them when I edit things.